packages/expo-image-picker/CHANGELOG.md

```diff
@@ -4,12 +4,20 @@
 
 ### 🛠 Breaking changes
 
+- remove typo workaround for `getMediaLibaryPermissionsAsync` method. ([#14646](https://github.com/expo/expo/pull/14646) by [@Simek](https://github.com/Simek))
+
 ### 🎉 New features
 
 ### 🐛 Bug fixes
 
+- add missing `cancelled` values for the Web responses. ([#14646](https://github.com/expo/expo/pull/14646) by [@Simek](https://github.com/Simek))
+
 ### 💡 Others
 
+- export missing types: `ImageInfo`, `ImagePickerMultipleResult`, `OpenFileBrowserOptions`, `ExpandImagePickerResult`, `UIImagePickerControllerQualityType` and `UIImagePickerPresentationStyle`. ([#14646](https://github.com/expo/expo/pull/14646) by [@Simek](https://github.com/Simek))
+- extract cancelled request object to separate type - `ImagePickerCancelledResult`. ([#14646](https://github.com/expo/expo/pull/14646) by [@Simek](https://github.com/Simek))
+- change `CameraPermissionResponse` type alias definition. ([#14646](https://github.com/expo/expo/pull/14646) by [@Simek](https://github.com/Simek))
```

Comment:
```suggestion
```
Looking at the code this change is not affecting user space at all 🤔 
It is directly connected with following change right?:
```diff
  import { PermissionResponse } from 'expo-modules-core';
- export { PermissionResponse as CameraPermissionResponse };
+ export type CameraPermissionResponse = PermissionResponse;
```

Reply:
Yes, it's just this line, but I want to be safe, than sorry! 😅 I will remove this line, no problem!

I have broken the `app-loading` export and no one catch this up before the release because of the missing changelog entry. :P